### ISSUE TITLE
Generate Interfaces from Query

### ISSUE THREAD

**Issue by brettjurgens** [avantcredit/gql2ts#30]
Allow an option to generate interfaces from a given query and schema

**Comment by danvk** [comment]
Apollo codegen does this: https://github.com/apollographql/apollo-codegen

```
apollo-codegen generate **/*.graphql --schema schema.json --target typescript --output schema.ts
```

**Comment by brettjurgens** [comment]
`apollo-codegen` writes directly to a file instead of allowing access to the value. i want to call a function and have it return the text. i looked into codegen a little while back and the file stuff was deeply engrained in it.

working on this in #46

**Comment by brettjurgens** [comment]
closed with #46
